ios/README.md

```diff
@@ -1,6 +1,5 @@
 # Trifle
 
-[![CI Status](https://img.shields.io/travis/CashApp/Trifle.svg?style=flat)](https://travis-ci.org/CashApp/Trifle)
```

Comment:
instead of removing this, lets link it to `https://github.com/cashapp/trifle/actions?query=workflow%3ACI+branch%3Amain` instead

Reply:
sure sounds good!